src/components/entity/feed-entry.vue

```diff
@@ -63,8 +63,8 @@ except according to the terms contained in the LICENSE file.
       <template v-else-if="entry.action === 'entity.update.version'">
         <span class="icon-pencil"></span>
         <span class="title">
-        <template v-if="entry.details.submissionCreate != null">
-          <i18n-t v-if="submission != null"
+        <template v-if="entry.details.submission != null">
```

Comment:
```suggestion
        <template v-if="submission != null">
```

I think it's a little clearer just to use `submission`, since `submission` is also used on the next line.